packages/connect-ui/src/lib/utils.ts

```diff
@@ -22,7 +22,12 @@ export function jsonSchemaToZod(schema: SimplifiedJSONSchema): ZodTypeAny {
     } else if (schema.format === 'email') {
         fieldString = fieldString.email();
     } else {
-        fieldString = fieldString.min(1, 'This field is required');
+        // Some providers takes empty default_value
+        // so we need to differentiate empty and not defined
+        // note that "optional" could work but does not exactly mean the same
```

Comment:
I want an `Option/Maybe` type. Please typescript, please! Maybe we do our own. It is probably 20 lines of code